ami/client/flowchart.py

```diff
@@ -219,6 +219,7 @@ def display(self, msg):
             elif self.ctrlWidget:
                 scrollarea = QtWidgets.QScrollArea(parent=self.win)
                 scrollarea.setWidget(self.ctrlWidget)
+                scrollarea.setWidgetResizable(True)
```

Comment:
Any box that defines a ctrlWidget should now have that widget take up the whole window and resize with the window.